Lib/lib2to3/main.py

```diff
@@ -154,6 +154,9 @@ def main(fixer_pkg, args=None):
                       help="List available transformations")
     parser.add_option("-p", "--print-function", action="store_true",
                       help="Modify the grammar so that print() is a function")
+    parser.add_option("--print-and-exec-function", action="store_true",
```

Comment:
We already have an option to make print a function ( `--print-function`) so this new `--print-and-exec-function` seems that is overlapping with that one.

Reply:
AFAIK there is no grammar only exec is a function but not print. So user have 2 options for selecting grammar, one is print is a function and exec is a statement the other one ise print and exec is a function.

Reply:
> AFAIK there is no grammar only exec is a function but not print. So user have 2 options for selecting grammar,

Sure, but that is an implementation detail, my point here is that it feels weird *as a user* to see those options. For example, that user could be confused on what happens if `--print-function` and  `--print-and-exec-function` are passed at the same time.

Reply:
Notice also that the docs say `Modify the grammar` not `select the grammar` so this adds to further confusion.

Reply:
Answer is simple, print is now seen as a function but exec is also seen as a function.

Reply:
> Answer is simple, print is now seen as a function but exec is also seen as a function.

Sure, and IMHO that shows that there is redundancy between the flags.

Reply:
but what if user only passes `--exec-function` and sees lib2to3 is implicitly considered print statements as functions

Reply:
> but what if user only passes `--exec-function` and sees lib2to3 is implicitly considered print statements as functions

That should not happen because my suggestion is that `--exec-function` should only affect `exec` and not `print`.

Reply:
So we are going to add another grammar

Reply:
It's two lines to `pygram.py`, no?

```
python_grammar_no_exec_statement = python_grammar.copy()
del python_grammar_no_exec_statement.keywords["exec"]
```

Reply:
yea I know, but I dont think this will be ever used. When I was adding `no_print_and_exec_statement` there was some linters already doing same thing. But it is your call, I'll fix this tonight

Reply:
Note: I edited my last comment with another suggestion (remove the keywords in `refactor.py` instead).

> yea I know, but I dont think this will be ever used. 

My whole point here is that this feels very inconsistent regarding the public API of the command line. Having `--no-option-a` and `--no-option-b-no-option-a` is suboptimal as the options should be `--no-option-a` and `--no-option-b`

Reply:
Looks cool to me.

Reply:
I should start using LCTM (looks cool to me) instead of LGTM :laughing:

Reply:
Ahaha :P